app/Http/Controllers/ExportController.php

```diff
@@ -213,22 +212,22 @@ public function exportExpansionLoot($expansionSlug, $fileType)
         }
 
         $subdomain = 'www';
-        if ($expansionId == 1) {
+        if ($expansionId === 1) {
             $subdomain = 'classic';
-        } else if ($expansionId == 2) {
+        } else if ($expansionId === 2) {
             $subdomain = 'tbc';
         }
 
-        $locale = '';
-        if (Illuminate\Support\Facades\App::getLocale() != 'en') {
-            if ($subdomain == 'www') {
-                $subdomain = '.' . Illuminate\Support\Facades\App::getLocale() . '.';
-            } else {
-                $locale = Illuminate\Support\Facades\App::getLocale() . '.';
+        $locale = App::getLocale();
+        if ($locale != 'en') {
+            $locale .= '.';
+
+            if ($subdomain === 'www') {
+                $locale = ".{$locale}";
             }

```

Comment:
The intention here is that we want to replace `www` with `de.` rather than append `de.`, so that we don't end up with `de.www.wowhead.com`.

That said, I see a different bug. It shouldn't be prefixed with a `.`.